### PR TITLE
regeneated examples #418

### PR DESCRIPTION
**Pull request by cdietrich**
regeneated examples #418
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
please review

File: org.eclipse.xtext.xtext.ui.examples/projects/org.eclipse.xtext.example.arithmetics.ui/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
+ org.eclipse.xtext.ui,
+ org.eclipse.xtext.ui.shared,
+ org.eclipse.xtext.ui.codetemplates.ui,
+ org.eclipse.ui.editors;bundle-version="3.5.0",
+ org.eclipse.ui.ide;bundle-version="3.5.0",
+ org.eclipse.ui,
+ org.eclipse.compare,
+ org.eclipse.xtext.builder,
+ org.eclipse.xtend.lib;resolution:=optional,
+ org.eclipse.xtext.xbase.lib;bundle-version="2.14.0"
```
Comment (kthoms) [inline org.eclipse.xtext.xtext.ui.examples/projects/org.eclipse.xtext.example.arithmetics.ui/META-INF/MANIFEST.MF]: Wurde in 2.14 schon API ergänzt, die genutzt wird?

Comment (cdietrich) [reply org.eclipse.xtext.xtext.ui.examples/projects/org.eclipse.xtext.example.arithmetics.ui/META-INF/MANIFEST.MF]: nein aber das wird so generiert

Comment (kthoms) [reply org.eclipse.xtext.xtext.ui.examples/projects/org.eclipse.xtext.example.arithmetics.ui/META-INF/MANIFEST.MF]: There is a property in the generator for that. Might have been replaced too greedy.

Comment (cdietrich) [reply org.eclipse.xtext.xtext.ui.examples/projects/org.eclipse.xtext.example.arithmetics.ui/META-INF/MANIFEST.MF]: yes but i think it should be in sync anyway

Comment (kthoms) [reply org.eclipse.xtext.xtext.ui.examples/projects/org.eclipse.xtext.example.arithmetics.ui/META-INF/MANIFEST.MF]: ok, might be safer

**Pull request merged**